packages/terra-menu/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+ * Added context for menu item selected/unselected state and sub menu keyboard navigation instructions.
```

Comment:
for consistency : 

```suggestion
* Added
  * Added context for menu item selected/unselected state and sub menu keyboard navigation instructions.
```

Reply:
Added space for doc file consistency - [3f88409](https://github.com/cerner/terra-framework/pull/1718/commits/3f884096957816abfff6d5bd308a580b440f6582)